src/app/components/organisms/cart_items/index.tsx

```diff
@@ -153,26 +152,16 @@ font-size: 14px;
 font-weight: 500;
 padding: 18.7px 14.11px 18.12px 24.75px;
 `;
-const notify = () => toast('product Removed',  {
-  duration: 4000,
-  position: 'top-center'});
  const CartItems = ()=> {
-  const {cartProducts, cartTotalQty, handleRemoveProduct} = useCart()
+  const {success} = useNotify()
+  const {cartProducts ,handleRemoveProduct, cartTotalQty, totalProductCount} = useCart()
   const removeProduct = (product: TProductTesting)=> {
-    notify()
     handleRemoveProduct(product)
+    success('Product Removed')
```

Comment:
```suggestion
    success('You have remove this product from your cart items')
```

Reply:
And I don7t think this is the best place to declare this success message. What if there was an error removing the product, the handling of error was not defined